requirements/pytorch/test.txt

```diff
@@ -16,4 +16,4 @@ fastapi <0.98.0  # for `ServableModuleValidator`
 uvicorn <0.22.1  # for `ServableModuleValidator`
 
 tensorboard >=2.9.1, <2.14.0  # for `TensorBoardLogger`
-protobuf <=3.20.1 # strict  # an extra is updating protobuf, this pin prevents TensorBoard failure
+protobuf <=3.20.3 # strict  # an extra is updating protobuf, this pin prevents TensorBoard failure
```

Comment:
This might not be necessary anymore as the comment suggests

```suggestion
```

Reply:
```suggestion
protobuf <=3.20.3
```